presto-main/src/test/java/io/prestosql/execution/TestSetSessionTask.java

```diff
@@ -167,6 +197,11 @@ private void testSetSessionWithValidation(Expression expression, String expected
         testSetSessionWithParameters("positive_property", expression, expectedValue, emptyList());
     }
 
+    private void testSetSessionWithEnum(Expression expression, String expectedValue)
```

Comment:
I think the tests will be easier to read if we add a parameter for property name to  `testSetSession` method. we won't require  `testSetSessionWithValidation` and `testSetSessionWithEnum` anymore. 

even now, the difference in `testSetSession` and `testSetSessionWithValidation` is only justified considering the behavior of property metadata that's defined completely outside of the method. Thoughts?